homeassistant/components/onkyo/config_flow.py

```diff
@@ -165,6 +167,50 @@ async def async_step_eiscp_discovery(
             ),
         )
 
+    async def async_step_ssdp(
+        self, discovery_info: ssdp.SsdpServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle flow initialized by SSDP discovery."""
+        _LOGGER.debug("Config flow start ssdp auto discovery: %s", discovery_info)
+
+        udn = discovery_info.ssdp_udn
+        if udn:
+            udn_parts = udn.split(":")
+            if len(udn_parts) == 2:
+                uuid = udn_parts[1]
+                last_uuid_section = uuid.split("-")[-1].upper()
+                await self.async_set_unique_id(last_uuid_section)
+                self._abort_if_unique_id_configured()
+
+        host = urlparse(discovery_info.ssdp_location).hostname
+
+        if not host:
+            _LOGGER.error("SSDP host is None")
+            return self.async_abort(reason="unknown")
+
+        try:
+            info = await async_interview(str(host))
+        except Exception:
+            _LOGGER.exception("Unexpected exception interviewing host %s", host)
+            return self.async_abort(reason="cannot_connect")
+
+        if info is None:
+            _LOGGER.debug("SSDP info is none: %s", host)
+            return self.async_abort(reason="cannot_connect")
+
+        await self.async_set_unique_id(info.identifier)
+        self._abort_if_unique_id_configured()
+
+        self._receiver_info = info
+        self.context.update(
+            {
+                "title_placeholders": {
+                    "name": discovery_info.upnp.get(ssdp.ATTR_UPNP_FRIENDLY_NAME, host)
```

Comment:
Maybe let's use `model_name`, since that is going to be the final title, so it will be less confusing?

Reply:
What if I use model name (friendlyName or ip) so that way it could clarify if you have multiple of the same reciever. IE "TX-NR6050 (Living Room Speaker)"

Reply:
OK. The question is how many people change the default friendly name. Maybe IP is better here, cause it is guaranteed to be unique.